sips/sip-021/sip-021-nakamoto.md

```diff
@@ -0,0 +1,562 @@
+
+If the PoX address holder votes for both "yes" and "no" by the end of the vote, the vote will be discarded.
+
+Note that this voting procedure does not apply to Stacking pool operators. Stacking pool operator votes will not be considered.
+
+#### Pooled Stacking
+
+If the user is stacking in a pool, then they must send a minimal amount of STX from their Stacking address to one of the following Stacks addresses to commit their STX to a vote:
+
+- For **"yes"**, the address is `SP00000000000003SCNSJTCSE62ZF4MSE`. This is the c32check-encoded Bitcoin address for "yes" (`11111111111111X6zHB1bPW6NJxw6`) above.
+- For **"no"**, the address is `SP00000000000000DSQJTCSE63RMXHDP`. This is the c32check-encoded Bitcoin address for "no" (`1111111111111117Crbcbt8W5dSU7`) above.
+
+From there, the vote tabulation software will track the STX back to the sender, and verify that the sender also has STX stacked in a pool. The Stacked STX will be tabulated as a "yes" or "no" depending on which of the above two addresses receive a minimal amount of STX.
+
+If the Stacks address holder votes for both "yes" and "no" by the end of the vote period, the vote will be discarded.
+
+## For Non-Stackers
+
+Users with liquid STX can vote on proposals using the [Ecosystem DAO](https://stx.eco).
+Liquid STX is the users balance, less any STX they have locked in PoX stacking protocol,
+at the block height at which the voting started (preventing the same STX from being transferred between accounts and used to effectively double vote).
+This is referred to generally as "snapshot" voting.
+
+For SIP 21 Nakamoto Upgrade to pass 66% of all liquid STX committed by voting
+must in favour of the proposal.
+
+### For Miners
+There is only one criterion for miners to activate this SIP: they must mine the Stacks blockchain up to and past the end of the voting period. In all reward cycles between cycle 75 and the end of the voting period, PoX must activate.
+
+### Examples
+
+#### Voting "yes" as a solo Stacker
+
+Suppose Alice has stacked 100,000 STX to `1LP3pniXxjSMqyLmrKHpdmoYfsDvwMMSxJ` during at least one of the voting period's reward cycles. To vote, she sends 5500 satoshis for **yes** to `11111111111111X6zHB1bPW6NJxw6`. Then, her 100,000 STX are tabulated as "yes".
+
+#### Voting "no" as a pool Stacker
+
+Suppose Bob has Stacked 1,000 STX in a Stacking pool and wants to vote "no", and suppose it remains locked in PoX during at least one reward cycle in the voting period. Suppose his Stacks address is `SP2REA2WBSD3XMVMYS48NJKS3WB22JTQNB101XRRZ`. To vote, he sends 1 uSTX from `SP2REA2WBSD3XMVMYS48NJKS3WB22JTQNB101XRRZ` for no to `SP00000000000000DSQJTCSE63RMXHDP`. Then, his 1,000 STX are tabulated as "no."
+
```

Comment:
Need to add "Activation Status" to log the voting results, similarly to [SIP-015 Stacks 2.1](https://github.com/stacksgov/sips/blob/main/sips/sip-015/sip-015-network-upgrade.md).
Below text is what I recommend: 

**Activation Status**
At the end of cycle 79, the following vote was calculated. A total of 177,166,148 STX participated.

* For solo stacking, 100% voted 'Yes.' Total voting power is 159,087,747 STX balance with votes cast from 28 accounts.
(Of the 159 million solo stacking voting power, StackingDAO’s internal voting contributed over 19 million STX in voting power.
    _-Total number of STX balance used to vote Solo Stacking by StackingDAO: 19,370,000.
    -6,827 StackingDAO accounts took part in the vote: 6,681 accounts voted in favor, 97.86%, 146 StackingDAO accounts voted against, 2.14%._)

* For pool stacking, 100% voted 'Yes.' Total voting power is 16,524,829 STX balance with votes cast from 370 accounts.

* For non-stackers, 99.98% voted 'Yes.' Total voting power of Yes is 1,553,301 STX balance with votes cast from 2,303 accounts. Total voting power of No is 271 STX balance from 16 accounts.



All voting criteria from STX holders have been met. A breakdown of the transactions can be found [here](https://stx.eco/dao/proposals/SP3JP0N1ZXGASRJ0F7QAHWFPGTVK9T2XNXDB908Z.bdp001-sip-021-nakamoto/results).

A copy of the scripts used to tabulate the solo and pool stacking can be found [here](https://github.com/stacksgov/sips/tree/main/sips/sip-021/nakamoto-votes.zip).

Reply:
If this gets approved, is it ready to merge into main repo? @whoabuddy @wileyj

Reply:
I just added your text to the bottom of the file 👍